.github/workflows/test-backend.yaml

```diff
@@ -0,0 +1,38 @@
+name: Test Backend
+
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "backend/**"
+      - "!backend/README.md"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        # Step into backend
+        working-directory: ./backend
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9.*"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry config virtualenvs.create false
+          poetry install --no-interaction --no-ansi
+
+      - name: Run tests
+        run: |
+          pytest
```

Comment:
I think this will need to change, based on yesterday's discussion (or we should commit the __init__)

Reply:
Because of the workflow trigger definition. (the `on.paths` section)

Reply:
We said we will just mark the `tests` dir as a regular pkg with an `__init__.py`. That way this line doesn't have to change.

Reply:
understood, thanks. Do we want changes in the `workflow/test-backend.yaml` file to trigger a CI?

Reply:
To make sure we are not breaking the testing completely. Yeah, that makes sense.

Reply:
@minottic it fails atm, because there are no tests to run. (exit code 5)

Reply:
I was about to write the same comment :). I think we can merge it as it is